src/engine-actions/wait-for-swap-commitment.spec.js

```diff
@@ -21,7 +22,8 @@ describe('wait-for-swap-commitment', () => {
       setTimeout(() => stream.emit('data', openInvoice), 1)
       setTimeout(() => stream.emit('data', openInvoice), 2)
       setTimeout(() => stream.emit('data', acceptedInvoice), 3)
-      expect(await waitForSwapCommitment(hash)).to.be.eql(acceptedInvoice)
+      expect(await waitForSwapCommitment(hash)).to.be.eql(
+        new Date(creationDate * 1000))
```

Comment:
[nit] would be nice to use real dates here rather than just replicating the implementation in the method.

Reply:
@treygriffith updated